docs/sources/administration/data-source-management/index.md

```diff
@@ -25,80 +25,52 @@ Before you can create your first dashboard, you need to add your data source.
 
 **To add a data source:**
 
-1. Select the cog icon on the side menu to show the configuration options.
-1. Select **Data sources**.
-
-   This opens the data sources page, which displays a list of previously configured data sources for the Grafana instance.
-
-1. Select **Add data source** to see a list of all supported data sources.
-
-   {{< figure src="/static/img/docs/v75/add-data-source-7-5.png" max-width="600px" class="docs-image--no-shadow">}}
-
-1. Enter the name of a specific data source in the search dialog.
-
-   You can also scroll through supported data sources grouped into time series, logging, tracing, and other categories.
-
-1. Move the cursor over the data source you want to add.
-
-1. Click **Select**.
-
-   This opens the data source configuration page.
-
+1. Click **Connections** in the left-side menu.
+1. Enter the name of a specific data source in the search dialog. You can filter by **Data source** to only see data sources.
+1. Click the data source you want to add.
 1. Configure the data source following instructions specific to that data source.
 
    For links to data source-specific documentation, see [Data sources]({{< relref "../../datasources" >}}).
 
 ## Data source permissions
 
-You can configure data source permissions to allow or deny certain users the ability to query a data source.
-Each data source's configuration includes a permissions page where you can enable permissions and restrict query permissions to specific **Users** and **Teams**.
+You can configure data source permissions to allow or deny certain users the ability to query or edit a data source. Each data source’s configuration includes a Permissions tab where you can restrict data source permissions to specific users, teams, or roles.
 
 > **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud).
 
-### Enable data source permissions
-
-By default, data sources in an organization can be queried by any user in that organization. For example, a user with the `Viewer` role can issue any possible query to a data source, not just
-queries that exist on dashboards they have access to.
-
-When permissions are enabled for a data source in an organization, the user who created the datasource can edit the datasource and in addition, viewers can query the datasource.
```

Comment:
Might be worth to keep some of this in. Maybe something like:

`By default, data sources in an organization can be queried by any user in that organization and edited by the user who created the data source and users with `Admin` role. For example, a user with the `Viewer` role can issue any possible query to a data source, not just queries that exist on dashboards they have access to.`

Reply:
It's actually still there but further down. But I'll add in the bit you've got about Admin roles as I think that's good too!